src/platform/packages/shared/kbn-esql-ast/scripts/generate_function_definitions.ts

```diff
@@ -304,13 +312,14 @@ function printGeneratedFunctionsFile(
     alias: ${alias ? `['${alias.join("', '")}']` : 'undefined'},
     signatures: ${JSON.stringify(signatures, null, 2)},
     locationsAvailable: [${locationsAvailable.join(', ')}],
+    ${license ? `license: '${license}',` : ''}${license ? '' : ''}
```

Comment:
```
license: '${licence}',
```

Why you just don't do this?  What is the result if ES doesnt have the license info?

Reply:
Mmm I see, ok this makes sense then